### PR TITLE
Fixed Bug in Which Mapper Can't Map to ListConfig if Input is None

### PR DESCRIPTION
**Pull request by NdubisiOnuora**
Example:
if value of config is the following
```python
ListConfig('example.food', {'item': 'name'})
```
And input is
```python
{'example': {'food': None}}
```

The following error is shown
`TypeError: 'NoneType' object is not iterable`

This fix just returns an empty list if the input is `None`.

### CODE REVIEW

**Comment by NdubisiOnuora** [comment]
@miki725, please review and merge.

File: simplepath/__init__.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 
 __author__ = 'Miroslav Shubernetskiy'
-__version__ = '0.3.4'
+__version__ = '0.3.5'
```
Comment (miki725) [inline simplepath/__init__.py:6]: can you update `HISTORY.rst` as well?

Comment (NdubisiOnuora) [reply simplepath/__init__.py:6]: Done

**Comment by NdubisiOnuora** [comment]
@miki725: whenever you're ready to re-review and merge.

File: HISTORY.rst [hunk 2]
```diff
@@ -3,19 +3,25 @@
 History
 -------
 
+0.3.5 (2018-06-14)
+~~~~~~~~~~~~~~~~~~~~~
+
+* Fixed bug in which TypeError would result if a None value were mapped to a ``ListConfig``
```
Comment (miki725) [inline HISTORY.rst]: wrap `TyperError` around double backticks to render as code. same for `None`

Comment (NdubisiOnuora) [reply HISTORY.rst]: Done

**Comment by NdubisiOnuora** [comment]
@miki725, all changes have been made.

**Comment by NdubisiOnuora** [comment]
@miki725: ping.

**Comment by NdubisiOnuora** [comment]
Yaissssssssssss!

**Pull request merged**